feature-group-definitions/compute-pressure.yml

```
@@ -0,0 +1,6 @@
+name: Compute Pressure API
```

```suggestion
name: Compute pressure
```

Or:

```suggestion
name: Pressure observer
```

---

I think the former "Compute pressure" or some third option like:

```suggestion
name: CPU compute pressure
```

But the discussion over in https://github.com/web-platform-dx/web-features/pull/762#issuecomment-2032969796 has me thinking that whatever we choose here might be short-lived, so I'm not fussed by any choice.